mh/server.py

```diff
@@ -206,19 +207,24 @@ def fileno(self):
         """
         return self.socket.fileno()
 
-    def serve_forever(self):
+    def serve_forever(self, broadcast_queue):
+        self.broadcast_queue = broadcast_queue
         self.__is_shut_down.clear()
         try:
             with self.selector as selector:
                 selector.register(self, selectors.EVENT_READ)
 
+                event_watch = Timer()
```

Comment:
It will do for the time being but a clock should be used. Having a single timer prevent handling multiple timed events.